docs/ref/contrib/admin/filters.txt

```diff
@@ -176,6 +176,18 @@ allows to store::
             ('title', admin.EmptyFieldListFilter),
         )
 
+It is also possible to specify a custom divider for list filters
+while using the ``__in`` lookup. By default ``__in`` lookup uses comma
+to separate different values.
+``django.contrib.admin.CustomDividerFilter`` can be used to specify a
+divider other than comma.
+You can specify a different custom divider as shown in the example below::
+
+    class FilterWithCustomDivider(admin.CustomDividerFilter):
+        def get_divider(self):
+            # return the custom divider as a string.
+            return '%'
```

Comment:
Just a thought - perhaps consider a different example character; % is already used for URL encoding (so `%` _can_ also be `%25`). It's probably not an _issue_ per se, mind.

Reply:
I agree. I will use a different character.